### DISCUSSION TITLE
Minimum Terraform version

### DISCUSSION THREAD

**Discussion by mdaleo404** [opentofu/opentofu#880] [General]
Hi all!

What specific minor version of Terraform should we be on before switching to OpenTofu? 1.5.5 or 1.5.6 or higher (or does it not matter)?

Thanks!

**Comment by cam72cam** [comment]
We forked from 1.5.6, so that is the ideal.  However I would be surprised if there are any significant issues in the migration path from Terraform 1.5.x / 1.6.x to OpenTofu 1.6.0.  There have been several changes that may need to be addressed, but that is all documented in the CHANGELOG.

**Reply by mdaleo404** [reply]
Great! Thank you @cam72cam
